cfme/tests/v2v/test_v2v.py

```diff
@@ -0,0 +1,64 @@
+"""Test to validate basic navigations.Later to be replaced with End-to-End functional testing."""
+import fauxfactory
+import pytest
+
+from cfme.utils.log import logger
+
+pytest.mark.ignore_stream('5.8', '5.9')
```

Comment:
**Required**

This is still getting collected against 5.8 in PRT.  This either needs to be included in a pytestmark list, or decorated on the individual test case.